src/lib/components/Timeline.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+    const timelineItems = [
+      { id: 1, text: "First Macintosh computer", date: "1984" },
+      { id: 2, text: "iMac", date: "1998" },
+      { id: 3, text: "iPod", date: "2001" },
+      { id: 4, text: "iPhone", date: "2007" },
+      { id: 5, text: "Apple Watch", date: "2015" }
+    ];
```

Comment:
Could we make these typed props so the timeline is updated automatically?

Reply:
done! thanks for feedback!